cluster/ci/config/centos-bazel-go-oc.yaml

```diff
@@ -0,0 +1,17 @@
+kind: ImageStream
+metadata:
+  name: centos-bazel-go-oc
+  namespace: openshift
+spec:
+  lookupPolicy:
+    local: true
+  tags:
+  - annotations: null
+    from:
+      kind: DockerImage
+      name: quay.io/coreos/centos-bazel-go-oc:latest
```

Comment:
What builds this?

Reply:
I'll commit it, it's currently a local Dockerfile-it's origin-release Dockerfile with bazel and oc added